src/Compilers/CSharp/Portable/CodeGen/EmitStatement.cs

```diff
@@ -1092,7 +1096,22 @@ private void EmitSwitchStatement(BoundSwitchStatement switchStatement)
                         var value = boundLabel.ConstantValueOpt;
                         Debug.Assert(value != null
                             && SwitchConstantValueHelper.IsValidSwitchCaseLabelConstant(value));
-                        labelsBuilder.Add(new KeyValuePair<ConstantValue, object>(value, label));
+
+                        // in debug we should be able to set breakpoints on individual case lables, 
```

Comment:
Removed the check for Debug

---
In reply to: [93700623](https://github.com/dotnet/roslyn/pull/16054#discussion_r93700623) [](ancestors = 93700623)